src/vector_topic_modeling/evaluation.py

```
@@ -0,0 +1,227 @@
+            else:
+                a_i = 0.0
+
+            b_i = float("inf")
+            for j, vecs_j in enumerate(cluster_vectors):
+                if i == j or not vecs_j:
+                    continue
+                mean_dist = sum(
+                    (1.0 - cosine_similarity(v, other_v)) for other_v in vecs_j
+                ) / len(vecs_j)
+                if mean_dist < b_i:
+                    b_i = mean_dist
```

Ruff (0.15.7)

[warning] Replace `if` statement with `b_i = min(b_i, mean_dist)`

Replace with `b_i = min(b_i, mean_dist)`

(PLR1730)